monkey/monkey_island/cc/resources/agent_configuration.py

```diff
@@ -24,7 +24,7 @@ def get(self):
     @jwt_required
     def post(self):
         try:
-            configuration_object = AgentConfigurationObject.from_json(request.data)
+            configuration_object = AgentConfigurationObject.from_mapping(request.json)
```

Comment:
If the request data type is json, flask automatically parse the data and puts it into request.json

Reply:
What's the behavior if the request data type is not JSON and we try and access `request.json`?

Reply:
400, raise `BadRequestError`